package.json

```diff
@@ -13,7 +13,6 @@
     "cover:watch": "nodemon --watch test/ --exec 'npm run test && npm run cover'",
     "cover:show": "open coverage/index.html",
     "cover:alls": "nyc report --reporter=text-lcov | coveralls",
-    "prebuild": "npm run test",
```

Comment:
Why was this removed?

Reply:
thats an accident, was experimenting with something will replace.